java/CMakeLists.txt

```diff
@@ -16,10 +18,12 @@ include_directories(SYSTEM ${JAVA_INCLUDE_PATH2})
 include_directories("src/main/native")
 include_directories("src/test/native")
 
+find_package(Arrow REQUIRED)
 find_package(gar REQUIRED)
 
 add_library(${LIBNAME} SHARED ${SOURCES})
 target_link_libraries(${LIBNAME} ${CMAKE_JNI_LINKER_FLAGS} gar)
+target_link_libraries(${LIBNAME} ${CMAKE_JNI_LINKER_FLAGS} Arrow::arrow_static)
```

Comment:
maybe link to shared library is better? otherwise it would take a long time to compile

Reply:
On one head, linking arrow as dynamic will leads error about libprotobuf: https://github.com/Thespica/GraphAr/actions/runs/6206365474/job/16850543098#step:6:9100

On the other head, I consider linking arrow as static just copy arrow lib once, rather than recompile, which only cost little time.

Reply:
OK, this can remain and open as a issue to fix in the future.